lib/msf/base/sessions/command_shell_unix.rb

```diff
@@ -0,0 +1,13 @@
+module Msf::Sessions
+
+  class CommandShellUnix < CommandShell
+    def initialize(*args)
+      self.platform = "unix"
+      super
+    end
+    def shell_command_token(cmd,timeout = 10)
+      shell_command_token_unix(cmd,timeout)
+    end
+  end
+
+end
```

Comment:
> No newline at the end of file

Might be worth modifying your editor to add this trailing new line :+1:

Reply:
Argh :facepalm: